projects/storefrontapp-e2e-cypress/cypress/helpers/checkout-flow.ts

```diff
@@ -89,7 +89,7 @@ export function fillAddressForm(shippingAddressData: AddressData = user) {
 
 export function chooseDeliveryMethod() {
```

Comment:
It makes sense to rename this function after changing its behavior.

Did you run all e2e tests as well? Everything passed?

Reply:
changed to `verifyDeliveryMethod` and verified regression